lms/templates/pdf_viewer.html

```diff
@@ -49,19 +49,19 @@
     <%static:js group='courseware'/>
   </head>
 
-  <body tabindex="1">
+  <body tabindex="0">
```

Comment:
Let's remove this `tabindex` altogether. It doesn't make sense to focus on the `body`.